example/src/A11y/A11y.jsx

```diff
@@ -0,0 +1,156 @@
+import React, { useRef } from 'react';
+
+import { SnapList, SnapItem, useVisibleElements, useScroll, useDragToScroll } from 'react-snaplist-carousel';
+
+import styles from './styles.module.css';
+
+const Item = ({ onClick, children, visible, ...props }) => (
+  <div
+    className={styles.item}
+    style={{
+      background: visible ? '#bce6fe' : '#cccccc',
+      cursor: visible ? 'default' : 'pointer',
+    }}
+    onClick={onClick}
```

Comment:
To be consistent with the A11y, shall we make this component respond to a keypress enter or space?

Reply:
Updated!